preferred-pm/index.js

```diff
@@ -21,6 +21,12 @@ module.exports = async function preferredPM (pkgPath) {
       version: '*'
     }
   }
+  if (await pathExists(path.join(pkgPath, 'bun.lockb'))) {
```

Comment:
lockfiles are checked in popularity order. I don't think bun is used more than pnpm.

Reply:
I moved this under the yarn/npm/pnpm checks but above the `findUp('pnpm-lock')` checks.